extension-base/ctpGcfWrapper.js

```diff
@@ -0,0 +1,57 @@
+// Google Cloud Functions
+
+module.exports = {
+  ctpGcfWrapper: function(fn) {
+    return function(req, res) {
+      fn(req.body, ctpGcfResponse(res), console.log);
+    }
+  }
+}
+
+function ctpGcfResponse(res) {
+  return {
+    pass: passGcf(res),
+    errors: errorsGcf(res),
```

Comment:
By the same token I am wondering how to align naming a bit. `pass` and `error` feels slightly off. What about `resolve` and `reject`? It's more aligned with `Promise.resolve` and `Promise.reject`. Then the "factories" would be called `createRejection(ctx)` and `createResolve`?

Reply:
I like `reject`. `resolve` not so much... if it's widely used in node-land, I guess it's a good idea to use it (but I wouldn't use it if I'd port this to Java, for example).

Going with `resolve`, it should probably unify both `pass` and `updates`. I'll give it a try.

Reply:
Yeah, I understand. You could maybe go with reject and accept as accept is a better term for what goes on?